src/Propel/Generator/Command/InitCommand.php

```diff
@@ -304,4 +283,23 @@ private function reverseEngineerSchema(OutputInterface $output, array $options)
 
         return $schema;
     }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return ConsoleHelperInterface
+     */
+    protected function createConsoleHelper(InputInterface $input, OutputInterface $output)
+    {
+        if (class_exists('\Symfony\Component\Console\Helper\QuestionHelper', true)) {
```

Comment:
A line comment above 'this class only exists in Symfony 3. So if it exists we use our ConsoleHelper3 to work with Symfony 3 correctly' or something.